.gitignore

```diff
@@ -78,6 +78,9 @@ typings/
 # nuxt.js build output
 .nuxt
 
+# Vercel build output
```

Comment:
Is this Vercel? Or just yarn?

Reply:
`yarn build` runs `ncc build` so I think it's a Vercel thing.